Observer/AfterConfigSaveObserver.php

```diff
@@ -209,6 +214,21 @@ private function getExistingWebhook()
         return ['id' => null,'active_events'=>null];
     }
 
+    function getWebhooks($count=2, $skip=2)
```

Comment:
@enishant can we fetch 10 webhooks as default in one request

Reply:
@abdulwahidsharief Updated count to 10 and skip 0